src/mscorlib/src/System/Globalization/TextInfo.cs

```diff
@@ -35,6 +35,12 @@ public partial class TextInfo : ICloneable, IDeserializationCallback
         //                        Internal Information                        //
         //--------------------------------------------------------------------//
 
+        private enum Tristate : byte
```

Comment:
A bool? isn't guaranteed to be assigned atomically, making it problematic for these cases where multiple threads may be involved.